generate-stackbrew-library.sh

```diff
@@ -65,62 +62,37 @@ join() {
 	echo "${out#$sep}"
 }
 
-for version in "${versions[@]}"; do
-	suite="$(basename "$version")"
+for version; do
+	# buster, bullseye, focal, etc
+	codename="$(basename "$version")"
+	# debian, ubuntu
 	dist="$(dirname "$version")"
-	dist="$(basename "$dist")"
-
-	versionAliases=( $suite ${aliases[$suite]:-} )
-
-	case "$dist" in
-		debian)
-			if debianSuite="$(
-				wget -qO- -o /dev/null "https://deb.debian.org/debian/dists/$suite/Release" \
-					| gawk -F ':[[:space:]]+' '$1 == "Suite" { print $2 }'
-			)" && [ -n "$debianSuite" ]; then
-				# "stable", "oldstable", etc.
-				versionAliases+=( "$debianSuite" )
-				if [ "$debianSuite" = 'stable' ]; then
-					versionAliases+=( latest )
-				fi
-			fi
-			;;
-
-		ubuntu)
-			if ubuntuVersion="$(
-				wget -qO- -o /dev/null "http://archive.ubuntu.com/ubuntu/dists/$suite/Release" \
-					| gawk -F ':[[:space:]]+' '$1 == "Version" { print $2 }'
-			)" && [ -n "$ubuntuVersion" ]; then
-				versionAliases+=( "$ubuntuVersion" )
-			fi
-			;;
-	esac
+
+	export version
+	suite="$(jq -r '.[env.version].suite' versions.json)"
+	versionAliases=( "$codename" "$suite" )
```

Comment:
If we're going to let it be empty in `versions.sh`, we need to handle that case here too:

```suggestion
	versionAliases=( "$codename" )
	suite="$(jq -r '.[env.version].suite // empty' versions.json)"
	if [ -n "$suite" ]; then
		 versionAliases+=( "$suite" )
	fi
```